### ISSUE TITLE
Hi-Res audio category and signaling

### ISSUE THREAD

**Issue by AlexKalopsia** [jeffvli/feishin#172]
## What do you want to be added?

I think it would be awesome if we had a way to quickly find Hi-Res audio (96kHz sample rate), either through a category in the sidebar, or via search parameters for Albums/Songs.

## Additional context

reference of how Audirvana does it

![image](https://github.com/jeffvli/feishin/assets/706110/0800508d-c363-4990-bd2e-95bf1d4b5fc1)

**Comment by isle9** [comment]
What about adding it as a filter?

**Comment by wildrun0** [comment]
Yeah I'm agree there should be Hi-Res / Lossless logo that informs album quality. It is surprising that this has not yet been introduced.
That's how I see it. Maybe logo should be smaller but you get the idea.

![1](https://github.com/jeffvli/feishin/assets/37260144/57664c04-70b6-408d-83c1-badf559dbf5a)

**Comment by AlexKalopsia** [comment]
> It is surprising that this has not yet been introduced.

Let's not forget this is still an enhancement and there have surely been more pressing/important things to address 🙏

**Comment by wildrun0** [comment]
> more pressing/important things to address

Yes, I totally agree. I apologize if my words sound selfish or aggressive. I didn't mean that
Author does a really good job. player is amazing even in the form in which it is now
